vllm/worker/model_runner.py

```diff
@@ -138,11 +141,16 @@ def __init__(
 
         # Lazy initialization
         self.model: nn.Module  # Set after load_model
-        # Set if the backend is flashinfer.
-        self.flashinfer_workspace_buffer: torch.Tensor
         # Set after load_model.
         self.lora_manager: Optional[LRUCacheWorkerLoRAManager] = None
 
+        # Flashinfer fields
+        self.paged_kv_indptr_tensor = None
+        self.paged_kv_indices_tensor = None
```

Comment:
```suggestion
        self.flash_infer_paged_kv_indices_tensor = None
```

Why don't we prepend flash_infer to variables?